.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Can this file be removed from the commit history? It's a local editor file I don't think we need to commit. (maybe add to `.gitignore`?)

Reply:
Tracked in #5